annotation/app/schemas/categories.py

```diff
@@ -43,7 +43,6 @@ class Config:
 
 
 class CategoryResponseSchema(CategoryInputSchema):
-    # To be removed
     parents: Optional[List[dict]] = Field(default=[])
```

Comment:
`parents` and `children` are not Optional, they cannot be None in response, always list